Makefile

```
@@ -0,0 +1,71 @@
+##
+## ----------------
+## General
+## ----------------
+##
+
+COMPOSE_EXEC_PHP=docker-compose exec api
+
+all: help
+
+help: ## Display this message
+	@grep -E '(^[a-zA-Z0-9_\-\.]+:.*?##.*$$)|(^##)' Makefile | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}' | sed -e 's/\[32m##/[33m/'
```

L'affichage de `make help` est très sympa

Il y a un espace superflu devant les sections qu'on peut retirer avec ceci:

```suggestion
	@grep -E '(^[a-zA-Z0-9_\-\.]+:.*?##.*$$)|(^##)' Makefile | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}' | sed -e 's/\[32m## /[33m/'
```

```diff
-  ----------------            
-  Logs                        
-  ----------------            
+ ----------------            
+ Logs                        
+ ----------------
```

---

Corrigé :heavy_check_mark: